sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -471,6 +471,45 @@ def test_binomial():
     assert expand_func(binomial(n, -2, evaluate=False)) == 0
     assert expand_func(binomial(n, k)) == binomial(n, k)
 
+def test_multinomial():
+    x = Symbol('x')
+    n = Symbol('n', integer=True)
+    nz = Symbol('nz', integer=True, nonzero=True)
+    k = Symbol('k', integer=True)
+    kp = Symbol('kp', integer=True, positive=True)
+    kn = Symbol('kn', integer=True, negative=True)
+    u = Symbol('u', negative=True)
+    v = Symbol('v', nonnegative=True)
+    p = Symbol('p', positive=True)
+    z = Symbol('z', zero=True)
+    nt = Symbol('nt', integer=False)
+    kt = Symbol('kt', integer=False)
+    a = Symbol('a', integer=True, nonnegative=True)
+    b = Symbol('b', integer=True, nonnegative=True)
+
+    assert multinomial(12, 5, 4, 3) == 27,720
+    assert multinomial(3, 2, 0, 1) == 3
+    assert multinomial(4, 2, 1, 1) == 12
+    assert multinomial(5, 2, 2, 1) == 30
+    assert multinomial(17, 4, 12, 1) == 30940
+    assert multinomial(149, 144, 1, 4) == 2859400145
+    assert multinomial(-5, -2, -2, -1) == 0
+    assert multinomial(-3, -2, 0, -1) == 0
+    assert multinomial(-12, -5, -4, -3) == 0
+    assert multinomial(-4, -2, -1, -1) == 0
+    assert expand_func(multinomial(n, 1)) == n + 2
+    assert expand_func(multinomial(n, 2)) == (n + 1)*(n + 2)/2
+    assert expand_func(multinomial(n, 2, 3)) == (n + 1)*(n + 2)*(n + 3)*(n + 4)*(n + 5)/12
+    assert multinomial(n, 3).expand(func=True) ==  n**3/6 + n**2 + 11*n/6 + 1
+    assert expand_func(multinomial(n, 3)) ==  (n + 1)*(n + 2)*(n + 3)/6
+    assert multinomial(kp, kp + 1) == gamma(2*kp + 2)/(gamma(kp + 1)*gamma(kp + 2))
+
+    from sympy import I
+    assert multinomial(I, Rational(-89, 8)) == gamma(-81/8 + I)/(gamma(-81/8)*gamma(1 + I))
+    assert multinomial(I, 2*I) == gamma(1 + 3*I)/(gamma(1 + I)*gamma(1 + 2*I))
+    assert multinomial(Rational(-7, 6), I) == gamma(-1/6)/(gamma(-1/6 - I)*gamma(1 + I))
+    assert multinomial((1+2*I), (1+3*I)) == gamma(3 + 5*I)/(gamma(2 + 2*I)*gamma(2 + 3*I))
+    assert multinomial(I, 5) == gamma(6 + I)/(120*gamma(1 + I))
```

Comment:
It looks like copy pasted from #18960

Reply:
It would be better you would have cherry-picked the commit if you want to reuse something from other branch/PR.

Reply:
I will make the changes.

Reply:
Could you please have a look at the issues travis ci throws?